src/config.ts

```diff
@@ -40,8 +46,26 @@ export const configConstants = {
     stage: "dev",
     prefix: "sls",
     localPort: 7071,
+    os: FunctionAppOS.WINDOWS,
+  },
+  dockerImages: {
+    node: {
+      "8": "DOCKER|microsoft/azure-functions-node8:2.0",
+      "10": "DOCKER|microsoft/azure-functions/node:2.0"
+    },
+    python: {
+      "3.6": "DOCKER|microsoft/azure-functions/python:2.0"
+    }
   },
```

Comment:
In Linux, currently we accepts `PYTHON|3.6`, `PYTHON|3.7`, `PYTHON|3.8`, `NODE|8`, `NODE|10` in the **LinuxFxVersion**, and we will convert these settings to the proper images for you.

Placing a list of MCR docker images have been deprecated in LinuxFxVersion setting.